pcfuncs/ipban/config.py

```diff
@@ -0,0 +1,19 @@
+# config.py
+from pydantic import BaseSettings
+
+
+class Settings(BaseSettings):
+    # Constants related to Azure Table Storage
+    storage_account_url: str = "https://pctapisstagingsa.table.core.windows.net/"
```

Comment:
I think I'd prefer url, table name, and LAW id to not have defaults here, and have them be supplied exclusively through the environment. That way, they fail fast on startup if they are not set in production, where we _must_ override them. 

I think the time/gb values are good with defaults.